src/util/bip32.rs

```diff
@@ -1116,5 +1117,50 @@ mod tests {
         assert_eq!("42", &format!("{}", ChildNumber::from_normal_idx(42).unwrap()));
         assert_eq!("000042", &format!("{:06}", ChildNumber::from_normal_idx(42).unwrap()));
     }
+
+    #[test]
+    fn schnorr_keypair() {
+        let secp = Secp256k1::new();
+        let xpriv = ExtendedPrivKey::from_str("xprv9s21ZrQH143K3QTDL4LXw2F7HEK3wJUD2nW2nRk4stbPy6cq3jPPqjiChkVvvNKmPGJxWUtg6LnF5kejMRNNU3TGtRBeJgk33yuGBxrMPHi").unwrap();
+        let pair = KeyPair::from_seckey_slice(&secp, &xpriv.private_key[..]).unwrap();
+        assert_eq!(xpriv.to_schnorr_keypair(&secp), pair);
```

Comment:
I don't see any value in this test? What is this test testing? It is a one-line function code that is repeated in the test. 

If you feel there is value in this, we can check that the x-only public keys from the keypairs are the same for now. After another point release, we can re-add this test. I don't think we are losing any features for the next release because of this.